### PR TITLE
fix: [#186092347] Business Name and Number of Employees no longer required in profile

### PR DESCRIPTION
**Pull request by aloverso**
### Ticket

[#186092347](https://www.pivotaltracker.com/story/show/#186092347)

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation, if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

**Review by cbrunefearless — APPROVED** [review]
lgtm

File: web/test/pages/profile/profile-starting.test.tsx [hunk 1]
```diff
@@ -948,34 +947,10 @@ describe("profile - starting business", () => {
...
-            legalStructureId: "limited-liability-company",
-          }),
-        });
-        renderPage({ business });
-        fireEvent.blur(screen.getByLabelText("Business name") as HTMLElement);
-
-        clickSave();
-        await waitFor(() => {
-          expect(
-            screen.getByText(Config.profileDefaults.fields.businessName.default.errorTextRequired)
```
Comment (jphechter) [inline web/test/pages/profile/profile-starting.test.tsx:971]: This really has no bearing on functionality at all, just kindof a house keeping thought. Given we don't need this error message any more, should we delete `businessName.default.errorTextRequired` from `profile.json`?

**Review by jphechter — APPROVED** [review]

File: web/test/pages/profile/profile-starting.test.tsx (hunk 1 above)
Comment (aloverso) [reply web/test/pages/profile/profile-starting.test.tsx:971]: Great catch!

Comment (aloverso) [reply web/test/pages/profile/profile-starting.test.tsx:971]: In the future you don't need to include "This really has no bearing on functionality at all, just kindof a house keeping thought."

That is a completely reasonable review comment and it _should_ block merging of this story.

**Review by somabadri — APPROVED** [review]
good to merge after addressing jp's comments.

**Pull request merged**